python_dependencies/Pipfile

```diff
@@ -30,6 +30,8 @@ jinja2 = "==2.11.2"
 gevent = "==20.5.0"
 gunicorn = "==20.0.4"
 kafka-python = "==2.0.1"
+greenlet = "==0.4.16"
```

Comment:
This one's a bit weird - if you examine the lock file you'll see that `greenlet` was already installed and all the hashes are the same. However, the new version of the sample service was stuck constantly restarting the workers with a warning about `greenlet` binary compatibility and explicitly installing `greenlet` made it start so *shrug*